app/views/appeals/appeal.haml

```diff
@@ -10,16 +10,16 @@
     .row
         .col-md-4
             %h4.strong What can be appealed?
-            %p If you have been caught cheating, your appeal will be denied.
-            %p Appeals exist in order to catch mistakes, not to argue about the rules, or your personal virtues.
+            %p Any punishment which you believe was issued incorrectly or which is too severe can be contested.
+            %p Appeals exist in order to catch mistakes, not to argue about the rules.
         .col-md-4
             %h4.strong How are appeals resolved?
-            %p The moderator who punished you will review the evidence, and remove the punishment if they find it was in error.
-            %p You can escalate an appeal to have it reviewed by a senior staff member, but this will delay the resolution.
+            %p The moderator who punished you will review the incident, and appeal the punishment if they find it was in error or reduce it in severity if they feel it was too harsh.
```

Comment:
should be `either appeal, or reduce the severity of, the punishment if they find it was in error or too harsh`

Reply:
No